pvlib/iam.py

```diff
@@ -748,3 +748,108 @@ def marion_integrate(function, surface_tilt, region, num=None):
         Fd = pd.Series(Fd, surface_tilt.index)
 
     return Fd
+
+
+def fedis(aoi, surface_tilt, n=1.5, n_ref=1.4585):
+    """
+    Determine the incidence angle modifiers (IAM) for direct, diffuse sky,
+    and ground-reflected radiation using the FEDIS transmittance model.
+
+    The "Fresnel Equations" for Diffuse radiation on Inclined photovoltaic
+    Surfaces (FEDIS) [1]_ is an analytical solution of diffuse transmission
+    based on the rigorous integration of an alternate form of the
+    Fresnel equations. The approach leads to a simple yet accurate
```

Comment:
The claim made here and in the source publication is misleading.  I suggest that the so-called "alternate form of the Fresnel" equations" be implemented in pvlib before finalizing the docstring for Fedis.

Reply:
Is the issue that the phrase "alternate form" implies equivalence when in fact the math relies on an approximation?  And what would the function signature for that implementation look like (inputs & outputs)?

Reply:
>  And what would the function signature for that implementation look like (inputs & outputs)?

Nevermind, just saw #1564